pycontw/pycon.tw
# Schedule API

## Types of changes


## Description
1. Add ScheduleAPIView based on src/events/views.py:95 : _ScheduleView_
   -  schedule_html : Indicate latest schedule_html
   -  schedule_day : settings.EVENTS_DAY_NAMES.items()

2. Document: https://hackmd.io/pio2HdWXRZiMPKXOANe-FQ
## Steps to Test This Pull Request
1. Log in to 'http://127.0.0.1:8000/admin'
2. Add test data to events/Schedule
3. Check 'http://127.0.0.1:8000/api/events/schedule/' for outcome

## Outcome
![image](https://user-images.githubusercontent.com/20749936/116812680-9a6f0f80-ab82-11eb-867f-e654f6b9d452.png)
